x/interchainqueries/types/expected_keepers.go

```diff
@@ -18,3 +19,12 @@ type BankKeeper interface {
 	SendCoinsFromAccountToModule(ctx sdk.Context, senderAddr sdk.AccAddress, recipientModule string, amt sdk.Coins) error
 	// Methods imported from bank should be defined here
 }
+
+type ContractManagerKeeper interface {
```

Comment:
I like this design of contract manager more than in the nearby PR. an idea: what do you think about naming the `contractmanager` module as simple `sudo`?

Reply:
Nearby PR is not actual anymore. 
Not sure that I understand your idea about renaming, because current module provides not only sudo functionality.

Reply:
the idea is simple: since the main idea of the ContractManager module is to store contracts sudo-related info, execute sudo calls to contracts and provide queries for getting this data, it's imo meaningful to rename it to Sudo: SudoKeeper, available via cli as `neutrond query sudo ...` etc. Maybe it even makes sense to split this module to two: `ContractManager` module with `HasContractInfo` and `AddContractFailure` methods, and a `Sudo` one with all the `Sudo...` methods. Out of this task scope. Wyt?

Reply:
Good idea, but I think that Sudo is too generic name and may be confusing. Also if we will split module into 2 different modules we will:
1. Move back to the previous state of the sudo helpers
2. We will have one module with ability to write to state and to query information from state, and another module without any interfaces to process messages or to query state. I mean this module will not have any ability to directly interact with cli on tx. Not sure that in this case we need to have another module for sudo, because previously we had this solution.

And current approach may help us to cover other contract related interactions in future.